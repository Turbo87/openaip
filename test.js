const fs = require('fs');
const path = require('path');

const openaip = require('./index.js');

describe('fixtures', () => {
  let fixtures = fs.readdirSync(path.join(__dirname, 'fixtures'));

  for (let fixture of fixtures) {
    test(fixture, () => {
      let filePath = path.join(__dirname, 'fixtures', fixture);
      let content = fs.readFileSync(filePath, 'utf8');
      let result = openaip.parse(content);

      expect(result).toMatchSnapshot();
    });
  }
});
