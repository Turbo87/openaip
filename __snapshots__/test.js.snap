// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fixtures de_asp.aip 1`] = `
Object {
  "version": "5280e1c1c34273664f8d8b884fc5c4c309b3e475",
}
`;

exports[`fixtures de_hot.aip 1`] = `
Object {
  "version": "1542404",
}
`;

exports[`fixtures de_nav.aip 1`] = `
Object {
  "version": "1746655",
}
`;

exports[`fixtures de_wpt.aip 1`] = `
Object {
  "version": "1773968",
}
`;
